docusaurus/react-docusaurus-dontent-docs.plugin.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
         lastVersion: 'current',
```

Comment:
Until we release v12-final, I think this should point to `11.x.x`

Reply:
But then we will have docs paths with v12 in them. Is that ok?

Reply:
yes, this is fine.

Reply:
Updated